CONTRIBUTING.md

```diff
@@ -36,4 +36,16 @@ Make sure you can build the code. Familiarize yourself with the project workflow
 
 If you wish to submit a new feature, please open an issue to discuss it with the project maintainers - don't open a pull request without first discussing whether the feature fits with the project roadmap.
 
-Tests must be provided for all pull requests that add or change functionality. 
+Tests must be provided for all pull requests that add or change functionality.
+
+## Building the Docs
+
+The docs are built separately from the source code. Building the docs requires Python 3 and pip. This is then used to install Sphinx and dependencies, which then enable `make` to build the site.
+
+For example, on Linux / within WSL:
+
+* `sudo apt install python3-pip`
```

Comment:
Probably worth mentioning that the pip packages directory needs to be added to PATH.